docs/api.rst

```diff
@@ -20,22 +20,37 @@ API
 .. _calibration_ref:
 
 
-:mod:`tedana.model`: Modeling TE-dependence
---------------------------------------------------
+:mod:`tedana.decay`: Modeling signal decay across echoes
+--------------------------------------------------------
 
-.. automodule:: tedana.model
+.. automodule:: tedana.decay
    :no-members:
    :no-inherited-members:
 
-.. autosummary:: tedana.model
-   :toctree: generated/
+.. autosummary:: tedana.decay
    :template: function.rst
+   :toctree: generated/
 
-   tedana.model.fitmodels_direct
+   tedana.decay.fit_decay
+   tedana.decay.fit_decay_ts
 
-   :template: module.rst
+.. currentmodule:: tedana
+
+.. _calibration_ref:
 
-   tedana.model.fit
+
+:mod:`tedana.combine`: Combining time series across echoes
```

Comment:
Here you have `combine`, but below you've replaced it with `model`. We should harmonize.

Reply:
I just reordered them based on when they occur in the workflow. It's possible that I missed some changes though, so I apologize if that's adding to the confusion.